src/main/java/seedu/duke/command/ListCommand.java

```diff
@@ -41,8 +41,8 @@ private String getList(Shelf list) {
     public void execute(Shelf list) throws EmptyListException {
```

Comment:
You missed this line. Please search for "list" in all your command classes.

Also, as discussed before, since we are taking in Shelf the command method, there is no need ask for Shelf again in the execute method.

Reply:
Created a separate issue for this